static/options.html

```diff
@@ -1,30 +1,13 @@
 <!DOCTYPE html>
-
 <html lang="en">
   <head>
     <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-
-    <link rel="stylesheet" href="styles/options.css" />
+    <title>Codeium options</title>
   </head>
-
   <body>
-    <div class="container">
-      <h1>Options</h1>
-
-      <p>
-        Edit telemetry settings
-        <a href="https://www.codeium.com/profile">at the Codeium Website.</a>
-      </p>
-      <br />
-      <br />
-      <p>Alternate login option</p>
-      <div style="flex-direction: row">
-        <button id="tokenpage" class="button">Open token page</button>
-        <button id="entertoken" class="button">Enter token</button>
-      </div>
-    </div>
-
+    <div id="codeium-options"></div>
     <script src="options.js"></script>
   </body>
-</html>
+</html>
```

Comment:
Merge/rebase on main and some additional CI checks will run. I think they might complain about missing `\n` here.

Reply:
Looks like we still need the merge/rebase (CI is still failing)